src/StreamJsonRpc/Exceptions/RemoteInvocationException.cs

```diff
@@ -83,5 +85,56 @@ public RemoteInvocationException(string? message, int errorCode, object? errorDa
         /// The default implementation of this method produces a <see cref="CommonErrorData"/> object.
         /// </remarks>
         public object? DeserializedErrorData { get; }
+
+        /// <inheritdoc/>
+        public override string ToString()
+        {
+            string result = base.ToString();
+
+            CommonErrorData? errorData = this.DeserializedErrorData as CommonErrorData;
+            if (errorData != null)
```

Comment:
These two lines could be written as "if (this.DeserializedErrorData is CommonErrorData errorData)", couldn't they?